assets/src/blocks/TakeActionBoxout/ImagePlaceholder.js

```diff
@@ -1,7 +1,7 @@
 import { ImagePlaceholderIcon } from '../../components/ImagePlaceholderIcon';
 
-export const ImagePlaceholder = ({children}) =>
-  <div className={'boxout-image-placeholder'}>
-    <ImagePlaceholderIcon width={20} height={20} fill={'#ffffff'}/>
-    {children}
-  </div>
+export const ImagePlaceholder = ( { children } ) =>
+  <div className={ 'boxout-image-placeholder' }>
+    <ImagePlaceholderIcon width={ 20 } height={ 20 } fill={ '#ffffff' } />
```

Comment:
Here also we could write `fill="#ffffff"`

Reply:
Done